resources/wikia/modules/iframeWriter.js

```diff
@@ -17,6 +17,7 @@ define('wikia.iframeWriter', [
 	 * @param {string} params.code - code to put into the iframe
 	 * @param {number} params.width - desired width of the iframe
 	 * @param {number} params.height - desired height of the iframe
+	 * @param {string} [params.classes] - custom class(es) of the iframe (coma-separated if more than one)
```

Comment:
[] means optional: 
http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values

On 03/31/2015 01:59 PM, Igor Rogatty wrote:
> In resources/wikia/modules/iframeWriter.js
> <https://github.com/Wikia/app/pull/6758#discussion_r27472393>:
>
>> @@ -17,6 +17,7 @@ define('wikia.iframeWriter', [
>>  	 * @param {string} params.code - code to put into the iframe
>>  	 * @param {number} params.width - desired width of the iframe
>>  	 * @param {number} params.height - desired height of the iframe
>> +	 * @param {string} [params.classes] - custom class(es) of the iframe (coma-separated if more than one)
>
> Why the square brackets? Are you sure they should be separated by coma
> and not by space?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/app/pull/6758/files#r27472393>.
>